src/Microsoft.Management.Configuration/Microsoft.Management.Configuration.idl

```diff
@@ -432,6 +432,11 @@ namespace Microsoft.Management.Configuration
 
         [contract(Microsoft.Management.Configuration.Contract, 3)]
         {
+            // The environment in which to process the configuration set.

```

Comment:
How is the configuration contract version defined? Is it tied to the schema version? (i.e. not tied to our releases since it's still in preview only?)